### ISSUE TITLE
Path separators issue in fs.readdir

### ISSUE THREAD

**Issue by dizlexik** [zeit/pkg#172]
I've been trying to get a Next.js project working with pkg and I've finally tracked down the issue to `fs.readdir` being called in `glob`.

On Windows locally each of these works: `fs.readdirSync('C:\\somedir')` and `fs.readdirSync('C:/somedir')`

But in my pkg Windows executable the second of these fails: `fs.readdirSync('C:\\snapshot')` and `fs.readdirSync('C:/snapshot')`

I receive this error message: `Error: ENOENT: no such file or directory, scandir 'C:\snapshot'`

**Comment by dizlexik** [comment]
I dug into this a little bit more and wrote a small hack that is working for me. It's just a workaround for now since I don't have the time to figure out the right way to do this and make a proper PR, but here's the change I made. Hopefully this will help someone fix it the right way.

In `/prelude/bootstrap.js` I made the following changes to get this to work on Windows:
```js
fs.readdir = function (path) {
  var isRoot = isRootPath(path);

  // <hack>
  if (windows) arguments[0] = path = path.replace(/\//g, '\\');
  // </hack>

  if (!insideSnapshot(path) && !isRoot) {
    return ancestor.readdir.apply(fs, arguments);
  }
  if (insideMountpoint(path)) {
    return ancestor.readdir.apply(fs, translateNth(arguments, 0, path));
  }

  var callback = dezalgo(maybeCallback(arguments));
  readdirFromSnapshot(path, isRoot, callback);
};
```
And I copied the same line to `fs.readdirSync`.

I'm hoping a fix for this can make it into the next version because I'm running this in CI and can't easily include my hack there. If I can be of anymore help please let me know. Thanks!

**Comment by igorklopov** [comment]
Thanks! Going to fix now

**Comment by igorklopov** [comment]
Fixed in `pkg@4.1.4`
